atlassian_jwt_auth/frameworks/common/asap.py

```diff
@@ -11,6 +13,7 @@
 def _process_asap_token(request, backend, settings, verifier=None):
     """ Verifies an ASAP token, validates the claims, and returns an error
     response"""
+    logger = logging.getLogger('asap')
```

Comment:
Is `asap` enough of context for people using this library to connect it to `atlassian-jwt-auth`? (btw, it's kinda confusing that the pip package and the repo name don't match at all)